shawmut/bin/auto_offd.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env
+
+import time
+from datetime import datetime
+from subprocess import call
+import os.path
+import urllib2
+import json
+
+SCHEDULER_INTERVAL = 5
+ADDRESSES = [ '10.0.1.16', '192.168.13.6' ]
```

Comment:
My IP is 192.168.13.9 -- I've set static IPs for both of our iPhones. I will add DNS entries later.